mobile/about.html

```diff
@@ -12,6 +12,57 @@ <h1>About this document</h1>
     <main data-contents="about">
       <section>
         <h2>Change History</h2>
+        <section>
+          <h3>September 2020</h3>
+          <p>The <a href="https://www.w3.org/2020/09/web-roadmaps/mobile/">September 2020</a> snapshot aadds new W3C community groups incubations, new W3C working group deliverables, and refreshes the status of other specifications. Changes made since the November 2019 snapshot:</p>
```

Comment:
```suggestion
          <p>The <a href="https://www.w3.org/2020/09/web-roadmaps/mobile/">September 2020</a> snapshot adds new W3C community groups incubations, new W3C working group deliverables, and refreshes the status of other specifications. Changes made since the November 2019 snapshot:</p>
```